src/services/core.js

```diff
@@ -255,63 +255,63 @@ const news = {
     {
       id: '1',
       title: 'We have some problems! In two lines, in two lines ',
-      text: 'Lorem ipsum text...',
+      text: '<p>Lorem ipsum text...</p><p>Lorem ipsum text </p>',
```

Comment:
Shouldn't this be in the component instead? 
I am assuming we don't want the api to respond with html paragraphs?

Reply:
From design perspective in this text section sometimes they include few paragrahs. Probably sometimes they would like to include links if one link at the end is not enough. This can also come from WYSIWYG. Than it is not bad idea to have such a possibility that these is prepared for html content. If it comes as a string we cover that case, if it comes as html we cover that as well...
I wouldn't add it as component as it is typical WYSIWYG the way I see it.

Reply:
ok, I just thought they would always be just one paragraph and there is a sperate field for urls. 
But yeah this is of course more more future proof for different scenarios.